src/views/Swap/Swap.vue

```diff
@@ -429,7 +429,7 @@ export default {
       return this.$route.query.source || null
     },
     showNoLiquidityMessage () {
-      return !this.market || BN(this.min).gt(this.max)
```

Comment:
why was the last part removed? it makes sure no liquidity message is displayed if there is not enough for a minimum swap